style.css

```diff
@@ -31,11 +31,11 @@
 }
 
 body {
-  font: 15px "Helvetica Neue",Helvetica,Arial,sans-serif;
+  font: 14px "Helvetica Neue",Helvetica,Arial,sans-serif;
```

Comment:
This pushes the number of characters on each line over a threshold for me.  I like the line spacing, but I think that this regresses readability.

Reply:
Hm. For me, 15px gives the type a colouring that's too dark (and harder to read), on OSX. I don't suppose you'd be open to making the body slightly narrower? Otherwise, I'll revert.

Reply:
This is safari, right?  I'll take a look.

Reply:
I think that the width is largely dictated by the monospaced elements on the page, that might mean changing the font size there to compensate.

Would `font-weight: 300;` work for you?  That causes OSX to use Helvetica Neue Light rather than Helvetica Neue, which I think looks much cleaner.  This doesn't make any apparent difference on Windows with Arial, so that's a clear win.